client/my-sites/email/email-providers-in-depth-comparison/comparison-table/index.tsx

```diff
@@ -1,21 +1,27 @@
-import { FunctionComponent } from 'react';
-import type { TranslateResult } from 'i18n-calypso';
+/* eslint-disable wpcalypso/jsx-classname-namespace */
 
-export type EmailProviderFeatures = {
-	header: string;
-	tools: TranslateResult;
-	storage: TranslateResult;
-	importing: TranslateResult;
-	support: TranslateResult;
-	selectCallback: () => void;
-};
+import { FunctionComponent } from 'react';
+import type {
+	ComparisonTableProps,
+	EmailProviderFeatures,
+} from 'calypso/my-sites/email/email-providers-in-depth-comparison/types';
 
-type ComparisonTableProps = {
-	emailProviders: EmailProviderFeatures[];
-};
+const ComparisonTable: FunctionComponent< ComparisonTableProps > = ( { emailProviders } ) => {
```

Comment:
Thought: in the long term it may be more flexible to use a data structure that exposes the provider name/slug as a key, which would also make it easier to use the provider slug as the key for each element.

Reply:
Very goint point, I'll keep that in mind for the next iteration.